src/bio.c

```diff
@@ -21,7 +21,7 @@
 
 #if !defined(WOLFSSL_BIO_INCLUDED)
     #ifndef WOLFSSL_IGNORE_FILE_WARN
-        #warning bio.c does not need to be compiled seperatly from ssl.c
+        #warning bio.c does not need to be compiled separately from ssl.c
```

Comment:
Looks like this spelling error is also in `wolfcrypt/src/evp.c:24`.

Reply:
My general rule about spelling is to only fix it when I am already making changes in a file. This makes it less tedious for anyone applying patches.